pkg/util/fs.go

```diff
@@ -40,6 +42,15 @@ type fs struct {
 	runner CommandRunner
 }
 
+// JoinPath joins the path elements always using the UNIX way. This function
+// should be used when constructing path that is valid for running in container.
+func JoinPath(elem ...string) string {
+	if runtime.GOOS == "windows" {
+		return path.Join(elem...)
+	}
+	return filepath.Join(elem...)
```

Comment:
What's the point in using `filepath.Join` if you always want to have the unix path, just use `path.Join` in that case. 
Currently I'm reading this as: if I'm on windows use `/`, otherwise use `os.Separator` which ends up being `/` as well. Am I missing something here?